taxi/views.py

```diff
@@ -1,12 +1,24 @@
 from django.contrib.auth.decorators import login_required
-from django.shortcuts import render
+from django.http import HttpResponseRedirect
+from django.shortcuts import render, get_object_or_404
 from django.urls import reverse_lazy
 from django.views import generic
 from django.contrib.auth.mixins import LoginRequiredMixin
 
+from .forms import DriverCreationForm, DriverLicenseUpdateForm, CarForm
 from .models import Driver, Car, Manufacturer
 
 
+@login_required
+def assign_driver(request, pk: int):
+    car = get_object_or_404(Car, id=pk)
+    if request.user in car.drivers.all():
+        car.drivers.remove(request.user)
+    else:
+        car.drivers.add(request.user)
+    return HttpResponseRedirect(f"/cars/{pk}/")
```

Comment:
Use reverse for this, no need to do this as path
Also will be cool if you rewrite this view to class